kura/org.eclipse.kura.wire.script.filter.localization/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,11 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Script Filter Localization Resource
+Bundle-SymbolicName: org.eclipse.kura.wire.script.filter.localization;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Bundle-Vendor: Eclipse Kura
+Bundle-License: Eclipse Public License v1.0
+Bundle-Category: Localization
+Fragment-Host: org.eclipse.kura.localization;bundle-version="1.0.0"
+Bundle-RequiredExecutionEnvironment: JavaSE-1.6
```

Comment:
Very minor: since the host runs on Java 8, I would expect Java 8 also for the fragment